pages/common/ajson.md

```diff
@@ -0,0 +1,24 @@
+# ajson
+
+> Executes JSONPath with JSON document.
+> More information: <https://github.com/spyzhov/ajson>.
+
+- Read JSON from a file and execute JSONPath:
+
+`ajson "$..json[?(@.path)]" example.json`
```

Comment:
I'd probably advise single quotes here instead of double quotes to avoid issues with those special characters. Also, perhaps we could use the `{{token_syntax}}` here and make `example.json` more consistent?

```suggestion
`ajson '{{$..json[?(@.path)]}}' {{path/to/file.json}}`
```